### ISSUE TITLE
[FEATURE] add decimals to increment field

### ISSUE THREAD

**Issue by Ilushock** [AllYarnsAreBeautiful/ayab-desktop#719]
Dear developers. The ratio of the stich width to its height is around 1.4 (usually)

This is usually calculated after measuring a test knitting sample.
The program has the ability to increase the image size in multiples of integers.
It would be great to set these values ​​with decimal places.

**Comment by jonathanperret** [comment]
Thanks you for your feedback. Which feature of AYAB are you referring to exactly? And does your report apply to AYAB 1.0.0  beta or an older version?

**Comment by jonathanperret** [comment]
OK, I suppose you’re talking about the **Stretch** operation that’s in AYAB 1.0.
It’s true that it’s very minimalistic at the moment and only allows integer multiples.
Extending this to decimal numbers sounds reasonable, although we’d have to be careful to disable anti-aliasing so that new colors aren’t introduced into the image.
We’ll take a look at this after 1.0 is out.
In the meantime I suggest you use any available image editor to do the stretching — that’s what I do personally anyway, because one often has to tweak some details in the image after it is stretched.

**Comment by Ilushock** [comment]
Everything is clear with other graphic editors. But then explain why you don’t want to connect AYAB with the DesignaKnit program?
There shouldn't be any problems here at all.

**Comment by jonathanperret** [comment]
Not sure how relevant this is to the current issue, but we’d absolutely like to connect AYAB to DesignAKnit. Some steps have already been taken in that direction, but it will likely take some more time before something concrete is achieved.

**Comment by jonathanperret** [comment]
Forgot to mention that if you are a DesignAKnit user, you can already design your patterns in DAK and import them into AYAB for knitting!

**Comment by Ilushock** [comment]
In manuals it is called "How to determime knitting gaudge".
![Gauge](https://github.com/user-attachments/assets/86fff620-627f-44c4-98a6-44d81995e361)
 All calculations depend on it. And the pattern is built based on these calculations.

**Comment by Ilushock** [comment]
if these proportions are not observed, the drawing will not look natural

**Comment by jonathanperret** [comment]
Yes, I know about stitches not coming out as squares in general. The actual ratio depends on the method used.
